tools/osbuilder/node-builder/azure-linux/uvm_install.sh

```diff
@@ -35,7 +35,8 @@ if [ "${CONF_PODS}" == "yes" ]; then
 	cp -a --backup=numbered "${UVM_DBG_MEASUREMENT_FILE_NAME}" "${UVM_PATH}"
 fi
 
-cp -a --backup=numbered "${IMG_FILE_NAME}" "${UVM_PATH}"
+cp -a --backup=numbered "${IMG_FILE_NAME}" "${UVM_PATH}/${IMG_FILE_NAME}"
+cp -a --backup=numbered "${IMG_DBG_FILE_NAME}" "${UVM_PATH}/${IMG_DBG_FILE_NAME}"
 
```

Comment:
Shouldn't we add a condition on `$MODE` here?

Reply:
We assume in uvm_install that both modes of UVM have been built. I suppose we could break up the makefile targets into `make deploy-uvm-debug` `make deploy-uvm-release` like we do for `make uvm`?

Reply:
No no ok this makes sense, same behavior as package_install.sh - please ignore :)